fastlane/Fastfile

```diff
@@ -13,15 +13,15 @@ ENV["DOWNLOAD_METADATA"]="./fastlane/download_metadata.swift"
 
 USER_ENV_FILE_PATH = File.join(Dir.home, '.simplenotemacos-env.default')
 
-WORKSPACE = 'Simplenote.xcworkspace'
-INTERNAL_SCHEME = 'Simplenote'
-APP_STORE_SCHEME = 'Simplenote-AppStore'
-BUILD_FOLDER = 'build'
+@WORKSPACE = 'Simplenote.xcworkspace'
+@INTERNAL_SCHEME = 'Simplenote'
+@APP_STORE_SCHEME = 'Simplenote-AppStore'
+@BUILD_FOLDER = 'build'
```

Comment:
This is not the syntax for global constants in ruby though:
 - all-uppercase (without any prefix char) = to declare a **constant**. Think `let` in Swift. If that constant is declared at the top level, it's a global one, if it's declared inside some scope (inside a function or whatever), it's a local one.
 - lowercase = a **variable**. Like `var` in Swift.
 - prefixed with `@` = **instance variable** (more especially the _backing storage_ of attributes in a ruby class, for which you usually have a corresponding `attr_reader` or `attr_accessor` to provide public accessors). Think `@foo` ≈ the private ivar you put in a `.h` in ObjC, and `attr_reader :foo` ≈ the associated public `@property foo` accessor for it)
 - _(There are other char-prefixes existing in ruby with different meanings, like `$` and `@​@`, but they are not relevant to the conversation here)_

Here you're mixing the two, and tbh I have no idea how ruby consider those cases. An instance variable `@` but declared outside of the scope of a `class` = ??

TL;DR: you could have kept the `WORKSPACE = ` without the `@` prefix all along. Since those constants are declared at top-level, there's no problem in referencing/using them anywhere else in the `Fastfile`, be it inside a lane or a function or wherever.

Reply:
Thanks @AliSoftware for the Ruby lesson. I looked for that information online but couldn't find a good post about it. I might have used a wrong query, though, because the mental model I had was not accurate.

As you may remember, [I originally extracted `WORKSPACE` and those other values using the lowercase syntax](https://github.com/Automattic/simplenote-macos/pull/870#discussion_r578371300), that is, as variables. I did that work as part of the original version of this branch, and when I tried to access them in the helper function, it didn't work. The only way to make it work in my flawed assumption that uppercase was just a convention rather than a language feature, was to make them instance variable, prefixing `@`, which in my mental model was (not sure why?) the equivalent of a global constant.

After applying your correction to make those values proper Ruby constants, I rebased the changes and kept the `@`; it didn't occur to me to check if it worked.

I know better now, thanks 🙇‍♂️

Reply:
I wanted to verify all that before posting it, and now that I wrote that code I can't let go of it. So here it is, my process in pictures.

First step: extract stuff, feel good about yourself, then wonder why Fastlane can't find the value.

![image](https://user-images.githubusercontent.com/1218433/110064887-17b81700-7dc2-11eb-8887-4c9938cac063.png)

Second step: make it an instance variable, because... not sure, maybe it worked in the past in other scripts I wrote for some reason? 🤷‍♂️ 

![image](https://user-images.githubusercontent.com/1218433/110064974-4930e280-7dc2-11eb-9f92-54afccc60220.png)

Third step: learn proper Ruby and have neat 🌯

![image](https://user-images.githubusercontent.com/1218433/110065015-61086680-7dc2-11eb-8683-0c7777c68ff2.png)